packages/fluentui/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -542,25 +552,46 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
           defaultProps: () => ({
             className: Dropdown.slotClassNames.triggerButton,
             content,
+            disabled,
             id: triggerButtonId,
             fluid: true,
             styles: styles.triggerButton,
             ...restTriggerButtonProps
           }),
           overrideProps: (predefinedProps: ButtonProps) => ({
             onClick: e => {
+              if (disabled) {
```

Comment:
The Button already has this in it's own onClick event.. I don't think it is necessary here